tests/feedback.test.ts

```diff
@@ -0,0 +1,195 @@
+  });
+
+  test('admins can view feedback from any user', async () => {
+    const conn = await DatabaseConnection.get();
+    const [user1, user2] = UserFactory.create(2);
+    const [admin] = UserFactory.with({ accessType: UserAccessType.ADMIN });
+    const [feedback1, feedback2] = FeedbackFactory.create(2);
+
+    const feedbackController = ControllerFactory.feedback(conn);
+    await new PortalState().createUsers([user1, user2, admin]).write(conn);
+
+    const submittedFeedback1Response = await feedbackController.submitFeedback({ feedback: feedback1 }, user1);
+    const submittedFeedback2Response = await feedbackController.submitFeedback({ feedback: feedback2 }, user2);
+    const allSubmittedFeedbackResponse = await feedbackController.getFeedback(admin);
+
+    expect(submittedFeedback1Response.feedback).toMatchObject(feedback1);
+    expect(submittedFeedback2Response.feedback).toMatchObject(feedback2);
+    expect(allSubmittedFeedbackResponse.feedback).toHaveLength(2);
+    expect(allSubmittedFeedbackResponse.feedback).toEqual(
+      expect.arrayContaining([submittedFeedback1Response.feedback, submittedFeedback2Response.feedback]),
+    );
+  });
+
+  test('members can view only their own feedback', async () => {
+    const conn = await DatabaseConnection.get();
+    const [user1, user2] = UserFactory.create(2);
+    const [feedback1, feedback2] = FeedbackFactory.create(2);
+
+    const feedbackController = ControllerFactory.feedback(conn);
+    await new PortalState().createUsers([user1, user2]).write(conn);
+
+    await feedbackController.submitFeedback({ feedback: feedback1 }, user1);
+    await feedbackController.submitFeedback({ feedback: feedback2 }, user2);
+    const user1Feedback = await feedbackController.getFeedback(user1);
+
+    expect(user1Feedback.feedback).toHaveLength(1);
+    expect(user1Feedback.feedback[0]).toMatchObject(feedback1);
+  });
+
+  test('admin can acknowledge and reward points for feedback', async () => {
+    const conn = await DatabaseConnection.get();
+    const [user] = UserFactory.create(1);
+    const [admin] = UserFactory.with({ accessType: UserAccessType.ADMIN });
+    const [feedback] = FeedbackFactory.create(1);
+    const status = FeedbackStatus.ACKNOWLEDGED;
+
+    const feedbackController = ControllerFactory.feedback(conn);
+    await new PortalState().createUsers([user, admin]).write(conn);
+
+    const submittedFeedbackResponse = await feedbackController.submitFeedback({ feedback }, user);
+    const { uuid } = submittedFeedbackResponse.feedback;
+    const acknowledgedFeedback = await feedbackController.updateFeedbackStatus(uuid, { status }, admin);
+
+    const persistedUser = await conn.manager.findOne(UserModel, { where: { uuid: user.uuid } });
+    const feedbackPointReward = Config.pointReward.FEEDBACK_POINT_REWARD;
+
+    expect(acknowledgedFeedback.feedback.status).toEqual(FeedbackStatus.ACKNOWLEDGED);
+    expect(persistedUser.points).toEqual(user.points + feedbackPointReward);
```

Comment:
Can we do this via the user controller? I think it's fine to use the simple GETs for different controllers.